src/plots/cartesian/index.js

```diff
@@ -198,6 +206,12 @@ exports.drawFramework = function(gd) {
     });
 };
 
+exports.rangePlot = function(gd, plotinfo, cdSubplot) {
+    makeSubplotLayer(plotinfo);
+    plotOne(gd, plotinfo, cdSubplot);
+    Plots.style(gd);
```

Comment:
Only tangentially relevant, but I'm not a *huge* fan of `Plots.style`. Long story short, when you transition the position and styles of an element at the same time, you *must* do them in a single `transition` operation, otherwise one discards the other.

In other words, for most d3 things that animate, I had to copy the styling into the inner join loop along with positioning instead of just applying styles on a completely separate pass (but to make sure it's robust and since the cost is probably small, I didn't remove styling from `module.style`).

It's fine as-is, just something to think about since the separate-pass styling approach wasn't fully compatible with what I needed.